### ISSUE TITLE
Make rsync related code not an agent

### ISSUE THREAD

**Issue by garagatyi** [codenvy/codenvy#1510]
Now our rsync related code is used as agents. We don't want this code to be agent, so we should appropriately refactor our code.

**Codenvy version:**    5.0.0

**Comment by TylerJewell** [comment]
@garagatyi - if we are going to look at some refactoring, we are starting to get requests from customers and users on the ability to customize the selection of what is rsync'd and restored from a workspace.  Similar to a unison profile - they are wanting to be able to apply includes and excludes using folder and file patterns. This may need another issue, but if we are thinking of a refactoring, we should think about adding in low level capabilities to allow for this filtering in the future.

**Comment by skabashnyuk** [comment]
depends on #1518

**Comment by TylerJewell** [comment]
rsync requirements no longer needed in k8s implementation
